deploy/testing-integration.yaml

```diff
@@ -12,6 +12,7 @@ objects:
       "ignore-check.kube-linter.io/no-liveness-probe": "probes not required on Job pods"
       "ignore-check.kube-linter.io/no-readiness-probe": "probes not required on Job pods"
   spec:
+    backoffLimit: 0
```

Comment:
I'm not huge fan of this btw... is there a plan to return to a reasonable value?